static/app/views/settings/project/server-side-sampling/modals/uniformRateModal.tsx

```diff
@@ -369,7 +369,7 @@ export function UniformRateModal({
             {
               learnMoreLink: (
                 <ExternalLink
-                  href={`${SERVER_SIDE_SAMPLING_DOC_LINK}getting-started/#2-set-a-uniform-sampling-rate`}
+                  href={`${SERVER_SIDE_SAMPLING_DOC_LINK}getting-started/#2-set-a-uniform-sample-rate`}
```

Comment:
this is currently wrong https://docs.sentry.io/product/data-management-settings/dynamic-sampling/getting-started/#2-set-a-uniform-sampling-rate